src/pages/terms-and-conditions/_business-grid.tsx

```diff
@@ -9,6 +9,7 @@ import PA from 'images/svg/terms/business-pa-tc.svg'
 import API from 'images/svg/terms/business-api-tc.svg'
 import BugBounty from 'images/svg/terms/bug-bounty.svg'
 import PDF from 'images/svg/regulatory/pdf-icon-black.svg'
+import { useCountryRule } from 'components/hooks/use-country-rule'
```

Comment:
Just update it to useRegion.
Rest LGTM.

Reply:
okay